spec/interactors/users/generate_access_token_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+describe Users::GenerateAccessToken do
+  describe ".call" do
+    user = User.create(
+      email: "user@admin.ru",
+      password: "123456"
+    )
+
+    let!(:interactor) { described_class.new(user: user) }
+
+    it "existince of token" do
```

Comment:
What it means?

Reply:
this means that the token will be checked for existence. Present or not

Reply:
i changes this description